migrations/versions/ff26585932ec_.py

```diff
@@ -0,0 +1,39 @@
+"""v3.7: migrate old reply_to email
+
+Revision ID: ff26585932ec
+Revises: fa07bd604a75
+Create Date: 2021-07-15 14:17:17.624748
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ff26585932ec'
+down_revision = 'fa07bd604a75'
+
+from alembic import op
+from sqlalchemy import orm
+from privacyidea.models import EventHandlerOption
+from privacyidea.lib.eventhandler.usernotification import NOTIFY_TYPE
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+    try:
+        for row in session.query(EventHandlerOption).filter(EventHandlerOption.Key == 'reply_to'):
+            reply_email = row.Value
```

Comment:
Can we please add a failsafe here to check, that the data is not migrated yet?
Otherwise we might really run into trouble if for any reason this script runs twice (since the DB is not stamped) and then a new configuration might be overwritten.